packages/Webkul/Admin/src/Resources/views/settings/roles/create.blade.php

```diff
@@ -161,7 +161,7 @@ class="primary-button"
                         </option>
                     </x-admin::form.control-group.control>
 
-                    <x-admin::form.control-group.error control-name="permission_type" />
+                    <x-admin::form.control-group.error control-name="permissions" />
```

Comment:
I believe we don’t need to make changes here because we are already using the error component for the related input field named `permission_type`. Therefore, the error component name should remain the same to specifically handle errors for this field.